tests/test_calculations.py

```diff
@@ -40,3 +40,18 @@ def test_melt_temp_2(): #Reference: http://biotools.nubic.northwestern.edu/Oligo
     oprimer = Primer("ATCG")
     basic_melt_temp = oprimer.melting_temperature(method=MeltingTemperature.BASIC)
     assert round(basic_melt_temp,1) == 12
+
+def test_single_runs_1():
```

Comment:
Add this reference for testcases:

`#Reference: https://www.oligoevaluator.com/OligoCalcServlet`